browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -186,7 +186,7 @@ EngineFlags ShouldBlockRequestOnTaskRunner(
 
   bool force_aggressive = SameDomainOrHost(
       ctx->initiator_url,
-      url::Origin::CreateFromNormalizedTuple("https", "youtube.com", 80),
+      url::Origin::CreateFromNormalizedTuple("https", "youtube.com", 443),
```

Comment:
Doesn't seem to matter, but I changed the port to 443 here for consistency.

Reply:
can you also please add a comment that links a ticket(s) in both cases